src/posts/posts.service.ts

```diff
@@ -12,71 +12,112 @@ export class PostsService {
     ) { }
 
     async getAll() {
-        return await this.postRepository.find();
+        try {
+            return await this.postRepository.find();
+        } catch (error) {
+            console.log(error);
+        }
+
     }
     async createPost(postData: PostDTO) {
-        const post = this.postRepository.create(postData);
-        await this.postRepository.save(post);
-        return post;
+        try {
+            const post = this.postRepository.create(postData);
```

Comment:
create là đủ k cần save nữa => return luôn : return await this.postRepository.create(postData);